5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
```

Comment:
I don't see any sense in having separate script for that, do this here, like we do in [sti-nodejs](https://github.com/openshift/sti-nodejs/blob/master/0.10/Dockerfile#L16-L19).